src/Context/TodosContext/TodosProvider.tsx

```diff
@@ -0,0 +1,67 @@
+import React, {
+  createContext, useState, useReducer,
+} from 'react';
+
+import { TodosListType, Todo } from '../../types/todosTypes';
+import { FiltersType } from '../../types/filterTypes';
+import { Actions } from '../../types/actionTypes';
+import { todosReducer } from '../TodosReducer';
+// import USER_ID from '../../helpers/USER_ID';
+// import { getTodos } from '../../api/todos';
+// import { loadTodosAction } from '../actions/actionCreators';
```

Comment:
remove that if you don't need it anymore

Reply:
Not fixed 😢